src/runtime/audience-activity-listener.js

```diff
@@ -76,23 +75,27 @@ export class AudienceActivityEventListener {
    */
   handleClientEvent_(event) {
     if (
-      this.storage_.get(Constants.USER_TOKEN) &&
-      event.eventType &&
-      audienceActivityLoggingEvents.has(event.eventType)
+      !event.eventType ||
+      !audienceActivityLoggingEvents.has(event.eventType)
     ) {
-      const pubId = encodeURIComponent(
-        this.deps_.pageConfig().getPublicationId()
-      );
-      const audienceActivityClientLogsRequest = this.createLogRequest_(event);
-      const url = serviceUrl(
-        '/publication/' +
-          pubId +
-          '/audienceactivitylogs' +
-          '&sut=' +
-          Constants.USER_TOKEN
-      );
-      this.fetcher_.sendBeacon(url, audienceActivityClientLogsRequest);
+      return;
```

Comment:
I think I'd prefer to wrap the below in the if check (closer to original) instead of returning for more clarity. In either case, a short comment making it explicit that checking first to avoid promise and a unit-test verifying promise not called would be great.